src/components/UiTooltip/index.tsx

```diff
@@ -1,55 +1,16 @@
-import {
-  ClickAwayListener,
-  ThemeProvider,
-  Tooltip,
-  Typography,
-  useMediaQuery,
-} from '@mui/material';
+import { ThemeProvider } from '@mui/material';
 import React from 'react';
 
 import { theme } from './theme';
+import WrapperUiTooltip from './TooltipWrapper';
 import { UiTooltipProps } from './types';
 
-function UiTooltip({
-  children,
-  title,
-  placement,
-  arrow,
-  sx,
-}: UiTooltipProps): React.ReactElement {
-  const [open, setOpen] = React.useState(false);
-  const isWideScreen: boolean = useMediaQuery(
-    '(min-width: 639px) and (max-width: 641px)'
-  );
-
-  React.useEffect(() => {
-    if (isWideScreen) {
-      setOpen(false);
-    }
-  }, [isWideScreen]);
-  function handleTooltipClose(): void {
-    setOpen(false);
-  }
-  function handleTooltipToogle(): void {
-    setOpen(!open);
-  }
-
+function UiTooltip(props: UiTooltipProps): React.ReactElement {
   return (
     <ThemeProvider theme={theme}>
-      <ClickAwayListener onClickAway={() => handleTooltipClose()}>
-        <Tooltip
-          open={open}
-          title={title}
-          placement={placement}
-          arrow={arrow}
-          sx={sx}
-        >
-          <Typography component="span" onClick={() => handleTooltipToogle()}>
-            {children}
-          </Typography>
-        </Tooltip>
-      </ClickAwayListener>
+      <WrapperUiTooltip {...props} />
```

Comment:
spread? what do we say about spread operators?

Reply:
as we said before, we will not use spread operators, that's why we add rule for this